solo/cli/_patches.py

```diff
@@ -119,7 +119,10 @@ def newParseUevent(uevent, desc):
             line = line.strip()
             if not line:
                 continue
-            k, v = line.split(b"=")
+            try:
+                k, v = line.split(b"=")
+            except ValueError:
+                pass
```

Comment:
Does it work if you change `pass` to `continue` in this case?

Reply:
yes, probably the better solution to skip the rest of the loop